xlive/Blam/Engine/math/matrix_math.cpp

```diff
@@ -130,48 +130,45 @@ void matrix4x3_from_point_and_vectors(real_matrix4x3* matrix, const real_point3d
 	return;
 }
 
-void matrix4x3_inverse(const real_matrix4x3* input, real_matrix4x3* output)
+real_matrix4x3* matrix4x3_inverse(const real_matrix4x3* input, real_matrix4x3* output)
```

Comment:
checked this function, in this instance the out vector pointer is not actually returned

Reply:
removed the return value